<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tremotedeskwindemoClass</class>
 <widget class="QWidget" name="tremotedeskwindemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1863</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tremotedeskwindemo</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_appkey">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roomId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_userId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>appKey:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>roomId:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>userId:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="login_">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leave_">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>leave</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="log_">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>1451</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tremotedeskwindemo.qrc"/>
 </resources>
 <connections/>
</ui>
